### PR TITLE
Bug 1118637 - Fix intermitent test failing on browser background r=kgrandon

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=708da45b391dc7f1ce41045fca983c8c3d67e818

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bc73e587abe038a632f921c12d2005535d819fca

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=85052ba642e149713ad052dab50a0c1fdfb7c525

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8bac650ca62140dab64ca04fa056b6954f2c7472

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fe731d0745ee2afe020c9c2751b46c10adc8226b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b36c21fbd0f2778fdca9136157e99b8c29106080

File: apps/system/test/marionette/browser_private_url_cleared_test.js [hunk 1]
```diff
@@ -34,9 +33,7 @@ marionette('Private Browser - URL Persistence', function() {
     // Use the home-screen search box to open up the system browser
     var url = server.url('sample.html');
     rocketbar.homescreenFocus();
-    search.triggerFirstRun(rocketbar);
-    rocketbar.enterText(url + '\uE006');
-    system.gotoBrowser(url);
```
Comment (KevinGrandon) [inline apps/system/test/marionette/browser_private_url_cleared_test.js]: Are you sure we should remove this?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b2fed981842652581e8d3d55b7ead676a263bb11

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=57714212eb941d6a5ab93676a6df74b1e2d58e9e

File: apps/system/test/marionette/browser_private_url_cleared_test.js (hunk 1 above)
Comment (albertopq) [inline apps/system/test/marionette/browser_private_url_cleared_test.js]: I'm not sure why we need to trigger a first rocketbar run.
At the other hand, system.goToBrowser switches the frame to that url, and I've seen plenty of those without actually never checking anything in the frame.

My impression is that those 2 were added (in several apps) in an attempt to mitigate the issue I'm fixing with this PR.

**Pull request merged**